projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.ts

```diff
@@ -32,6 +33,7 @@ import { CheckoutStepService } from '../../services/index';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ReviewSubmitComponent {
+  iconTypes = ICON_TYPE;
```

Comment:
line 176: need to add the formattedPrice from delivery cost in the `getDeliveryModeCard`

Reply:
in the spec, remove the featuremodule and config from the imports/providers